cmd/minishift/cmd/status.go

```diff
@@ -37,13 +39,15 @@ var statusFormat = `Minishift:  {{.MinishiftStatus}}
 Profile:    {{.ProfileName}}
 OpenShift:  {{.ClusterStatus}}
 DiskUsage:  {{.DiskUsage}}
+CacheUsage: {{.CacheUsage}} Bytes
```

Comment:
We would like to use the HumanReadable form as used in other places in our codebase.

Reply:
Thanks for the review, I will update and push the code to have MB